maven-core/src/test/java/org/apache/maven/project/ProjectBuilderTest.java

```diff
@@ -234,7 +234,7 @@ void testReadInvalidPom() throws Exception {
 
         // single project build entry point
         Exception ex = assertThrows(Exception.class, () -> projectBuilder.build(pomFile, configuration));
-        assertThat(ex.getMessage(), containsString("Received non-all-whitespace CHARACTERS or CDATA event"));
+        assertThat(ex.getMessage(), containsString("expected START_TAG or END_TAG not CHARACTERS"));
```

Comment:
END_TAG, not
(add comma)

Reply:
This is a test, so I can't change the message here.  I'd be happy to raise a follow-up as this message is sent by the generated xml parsers.